ResearchKit/Common/ORKHTMLPDFPageRenderer.h

```diff
@@ -0,0 +1,19 @@
+//
+//  ORKHTMLPDFPageRenderer.h
+//  ResearchKit
+//
+//  Created by Ortman, Chris E on 8/10/16.
+//  Copyright © 2016 researchkit.org. All rights reserved.
+//
+
```

Comment:
2 new lines before and after import block as per RK Coding Guidlines.

Add extra new line.